### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by YuriyBilskiy**
[DEMO LINK](https://yuriybilskiy.github.io/react_todo-app/)

### CODE REVIEW

File: src/components/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+          </span>
+
+          <ul className="filters">
+            <li>
+              <a
+                onClick={() => handleFilterChange(Status.All)}
+                href="#/"
+                className={
+                  filterStatus === Status.All ? 'selected' : ''
+                }
```
Comment (IvanFesenko) [inline src/components/Footer.tsx:40]: Use classnames library for such cases, fix all similar

File: src/components/Main.tsx [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+                      aria-label={`Delete todo: ${todo.title}`}
+                      onClick={() => removeTodo(todo.id)}
+                    />
+                  </div>
+                  <input
+                    type="text"
+                    className="edit"
+                    value={editedText}
+                    onChange={handleEditChange}
+                    onKeyDown={(e) => handleEditKeyDown(e)}
```
Comment (IvanFesenko) [inline src/components/Main.tsx:145]:
```suggestion
                    onKeyDown={handleEditKeyDown}
```

**Review by IvanFesenko — CHANGES_REQUESTED** [review]
- `Clear completed` button do not work

File: src/components/Main.tsx [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+        return todo;
+      });
+
+      setTodos(changeTodo);
+      setIsEditing(false);
+      setEditingTodoId(null);
+    } else if (event.key === 'Escape') {
+      setIsEditing(false);
+      setEditingTodoId(null);
+    }
```
Comment (VitaliyBodnar) [inline src/components/Main.tsx:94]:
```suggestion
      setIsEditing(false);
      setEditingTodoId(null);
```

File: src/components/Main.tsx [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+  };
+
+  const handleEditChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setEditedText(event.target.value);
+  };
+
+  const handleEditKeyDown = (
+    event: React.KeyboardEvent<HTMLInputElement>,
+  ) => {
+    if (event.key === 'Enter') {
```
Comment (VitaliyBodnar) [inline src/components/Main.tsx:79]: It's better to move "Enter" to the enum as it will make your code cleaner

**Review by VitaliyBodnar — CHANGES_REQUESTED** [review]

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Great work! The clear completed button doesn't work, consider fixing it

**Comment by YuriyBilskiy** [comment]
> Great work! The clear completed button doesn't work, consider fixing it

everything works for me

**Review by VitaliyBodnar — APPROVED** [review]
LGTM

**Pull request closed without merging**